### PR TITLE
Add linux-{aarch64, ppcle}, build from scr on windows, licenses

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
@conda-forge/kubernetes-helm ready for review / merge. thanks!

**Review by dhirschfeld — APPROVED** [review]

**Comment by dhirschfeld** [comment]
Great work - thanks @dbast

File: recipe/meta.yaml [hunk 1]
```diff
@@ -2,48 +2,51 @@
 {% set goname = "k8s.io/" + name %}
 
 {% set version = "3.16.2" %}
-{% set pkg_src = ('src/'+goname).replace("/", "\\" if win else "/") %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
```
Comment (jakirkham) [inline recipe/meta.yaml:5]: It looks like `goname` is only used in the `pkg_src` line

Also have found the following trick useful when constructing platform dependent paths 

```suggestion

{% set version = "3.16.2" %}
{% set pkg_src = os.sep.join("src", "k8s.io", name) %}
```

**Comment by jakirkham** [comment]
Thank you both! 🙏

**Comment by dhirschfeld** [comment]
Thanks @jakirkham! There's no automerge here (yet) so I'll try to incorporate that simplification in the next update!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -2,48 +2,51 @@
...
-    - make build  # [unix]
-    - make test-unit  # [unix and not arm64]
-    - pushd bin  # [unix]
-
+    - export USER=$(id -u -n)                                           # [osx]
+    - make build
+    - make test-unit                                                    # [unix and not arm64]
+    - go-licenses save ./cmd/helm --save_path ../../../license-files
+    - pushd bin                                                         # [unix]
+    - cd bin                                                            # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:25]: Both UNIX and Windows have `pushd` and `cd`

So this could be

```suggestion
pushd bin
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -2,48 +2,51 @@
...
-  number: 0
+  number: 1
   script:
     - pushd {{ pkg_src }}
-    - export USER=$(id -u -n)  # [osx]
-    - make build  # [unix]
-    - make test-unit  # [unix and not arm64]
-    - pushd bin  # [unix]
-
+    - export USER=$(id -u -n)                                           # [osx]
```
Comment (jakirkham) [inline recipe/meta.yaml:20]: Would recommend a comment for this line. Recognize it was already here. Still would be good to explain why this is needed to future readers

**Comment by dbast** [comment]
@jakirkham Thanks for your improvement ideas. We are getting there via next PR.

There is also some additional work to be done (potentially patching the Makefile) so that the version reported by `helm version` is correct ... currently (since before this PR) `version.BuildInfo{Version:"v3.16+unreleased", GitCommit:"3af2d7da7d9f2e85a25bfac84bb0364a4681cfe8", GitTreeState:"dirty", GoVersion:"go1.23.3"}`

**Pull request merged**